block/config.ml

```diff
@@ -2,9 +2,10 @@ open Mirage
 
 let main = foreign "Unikernel.Main" (console @-> block @-> job)
 
-let img = match get_mode () with
-  | `Xen -> block_of_file "xvda1"
-  | `Unix | `MacOSX -> block_of_file "disk.img"
+let img =
+  if_impl Key.is_xen
+    (block_of_file "xvda1")
+    (block_of_file "disk.img")
```

Comment:
It is available, but deprecated and people should not use it in the `config.ml` anymore.